common/web/keyman-version/package.json

```diff
@@ -20,7 +20,7 @@
   "license": "MIT",
   "type": "module",
   "devDependencies": {
-    "@types/node": "^18.7.13",
+    "@types/node": "^20.4.1",
```

Comment:
I see we're only updating "@types/node" for Developer-related components, rather than across the whole repo.  I assume that's to be done separately for the KMW components?

Reply:
Yeah, I think that belongs as a separate PR.